### DISCUSSION TITLE
Accessibility Conformance Report

### DISCUSSION THREAD

**Discussion by hoze1979** [stephanrauh/ngx-extended-pdf-viewer#2008] [General]
Would it be possible to get an accessibility conformance report from the team on the ngx-extended-pdf-viewer component to determine its Section 508 compliance? Thank you!

**Comment by stephanrauh** [comment]
Well... the team consists of a single person maintaining this project in their leisure time. I don't oppose your request - quite the contrary, I think it makes sense. It's just that I don't have enough spare time to do it.

However, you seem to be familiar with the topic. Can I talk you into getting the report and telling me what needs to be done to get it?

I know that's an unsatisfying answer - but then, the idea of open source is everybody contributes.
